main.tf

```diff
@@ -91,25 +90,28 @@ resource "aws_security_group" "instance" {
 resource "aws_instance" "example" {
   ami                    = "ami-0889a44b331db0194"
   instance_type          = "t2.micro"
+  # List of security group IDs to associate with. 
   vpc_security_group_ids = [aws_security_group.instance.id]
-
+  # User data to provide when launching the instance. 
+  # Updates to this filed will trigger a start/stop of 
+  # EC2 instance by default. 
   user_data = <<-EOF
               #!/bin/bash
               echo "Hello, World" > index.html
               nohup busybox httpd -f -p 8080 &
               EOF
-
+  #
   user_data_replace_on_change = true
 
   tags = {
     Name = "terraform-example"
   }
 }
 
-# Line 93 is used to create an index.html file and with 
+# Line 97 is used to create an index.html file and with 
```

Comment:
Move the comment to the appropriate line.

Reply:
When I move the lines it changes the color if when I put #.

Reply:
I moved it on top.